src/ralph/cli.py

```diff
@@ -547,8 +553,15 @@ def convert(from_, to_, ignore_errors, fail_on_unknown, **conversion_set_kwargs)
         click.echo(event)
 
 
+read_backend_types = [
```

Comment:
Why moving this here ? Same for `write` and `list` backends

Reply:
Moving it there so we stay DRY between what we give to `backends_options` for the CLI and what we use to retrieve `backend_type`. I agree that defining it as globals is not ideal!